cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,10 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
+		set:  SetBool,
```

Comment:
About this, I noticed an inconsistency of these options... dashed and cased... is that something we did?